### DISCUSSION TITLE
videoconf room credentials

### DISCUSSION THREAD

**Discussion by istvan60** [edumeet/edumeet#854] [Ideas]
Dear Developer,

I suggest bonding the creation of the videoconference rooms to e-mail addresses. This way only that person could lock the room who created it. He will just get a confirmation link, do you really want to lock the room.  With the same system the room creator could get a mail every two days asking, whether she/he wants to keep the room or not and without a reply the given room will be closed, i.e. can be used by anyone else.

Just an idea, great work btw!

**Comment by havfo** [comment]
Thank you for your interest in the project, and thank you for the input. We are currently looking at ways to handle management of rooms, and improving locking so this is hopefully something that will be better soon.

**Comment by misi** [comment]
@istvan60 It is almost done check the actual develop branch and user roles and user mapping function. We are working on the room locking actually.
